src/SonarScanner.MSBuild.PreProcessor/Roslyn/Model/Plugin.cs

```diff
@@ -19,12 +19,11 @@
  */
 
 using System;
-using System.Xml.Serialization;
 
 namespace SonarScanner.MSBuild.PreProcessor.Roslyn
 {
     /// <summary>
-    /// XML-serializable data class for a single SonarQube plugin containing an analyzer
+    /// Data class for a single SonarQube plugin containing an analyzer
```

Comment:
This class was never actually serialized, so I've updated the comment and removed the serialization attributes.